pandas/_libs/lib.pyi

```diff
@@ -1,6 +1,6 @@
 # TODO(npdtypes): Many types specified here can be made more specific/accurate;
 #  the more specific versions are specified in comments
-
+from decimal import Decimal
```

Comment:
Can this go inside a `if TYPE_CHECKING` block?

Reply:
tried this and the linter said TYPE_CHECKING in pyi files is unnecessary